createdisk.sh

```diff
@@ -39,7 +42,7 @@ EOF
 # https://bugzilla.redhat.com/show_bug.cgi?id=1956739
 ${SSH} core@${VM_IP} -- 'touch ~/.ssh/authorized_keys && chmod 0600 ~/.ssh/authorized_keys'
 
-# Shutdown and Start the VM after installing the hyperV daemon packages.
+# Shutdown and Start the VM after installing the hyperV daemon and removing moby packages.
```

Comment:
`after modifying the set of installed packages` or such? This way we don't have to keep it up to date?